bittensor/core/subtensor.py

```diff
@@ -1915,6 +1916,7 @@ def commit_weights(
 
         return success, message
 
+    # Community uses this method
```

Comment:
Should we include this comment in the code? If so, perhaps something more subtle like “function used for XYZ purpose”

Reply:
> Should we include this comment in the code? If so, perhaps something more subtle like “function used for XYZ purpose”

I don't think so we need to move the comments like this into the methods/functions. I keep it outside especially for us/developers. It means that marked method is used for backwards compatibility  only (not only within bittensor). 
I have a plan to delete all of same comments when SDK is ready for release.

Reply:
ok sounds good